.github/actions/backend-deploy-job/files/cloudbuild.yaml

```diff
@@ -98,7 +99,7 @@ steps:
             yq e '.spec.template.metadata.annotations += {"run.googleapis.com/vpc-access-egress": "private-ranges-only", "run.googleapis.com/vpc-access-connector": env(VPC_CONNECTOR)}' -i "$temp_file"
         fi
 
-        yq e '.spec.template.spec.containers[0].env += env(VAL)' -i "$temp_file"
+        yq e '.spec.template.spec.template.spec.containers[0].env += env(VAL)' -i "$temp_file"
```

Comment:
is this path correct? looked at a random cloudrun yaml, seems smth like this: 
spec:
  template:
    spec:
      containers:
        - name: my-container
          env:
            - name: VAL
               value: "some-value"

Reply:
Jobs's path is different than service. 

![image](https://github.com/user-attachments/assets/df5d82fc-3d5c-4f4f-8049-b15e495aaf5b)